ci/azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+trigger:
+- ms-iot
+
+jobs:
+  - job: Build67
```

Comment:
Is there anything special about the value "Build67"  I see you use it here and in the U-Boot yml file.

Reply:
A hold over from the iotcore repo. Build8 is for the i.MX8 build. I'm assuming that for all shared components we will probably want to separate the 6/7 and 8 builds.

Those are just internal tags to identify the job, the display name is what is normally shown.